backend/src/test/kotlin/tools/vitruv/vitruvAdapter/vitruv/api/CreateWindowContentMockupTest.kt

```diff
@@ -0,0 +1,58 @@
+package tools.vitruv.vitruvAdapter.vitruv.api
```

Comment:
imo the naming convention ClassNameTests.kt and methodNameTests() are more readable